Automattic/themes
# Convert line-height and font-size into CSS custom properties

This PR adds support for tweaking the line-height and font-size using Global Styles. Goes with https://github.com/Automattic/wp-calypso/pull/38754

![2001-global-styles-font-settings-14-1218](https://user-images.githubusercontent.com/583546/72343539-680c7b00-36cf-11ea-82a8-d846fef2b4fd.gif)

**Some thoughts:**

- I've found a few instances where invalid values were used and changed them to use the body line-height instead (see comments on code).
- There are a few other places that use a number instead of a SASS variable. I've left those untouched, but I wonder whether we need to modify them to react to user choices.
- I was thinking that, for release, we should also add font sizes as a minimum and potentially letter-spacing as well. What do you think? Is tweaking line-height useful enough?

#### Testing instructions

* Follow instructions on https://github.com/Automattic/wp-calypso/pull/38754
* From the root directory, run `cd varia && npm install && npm run build && npm run build` (twice, so rtl stylesheet compile as well).
* Copy the `varia` dir into the `wp-content/themes` directory of a WordPress instance.
* Switch to varia as the theme active and open a post.
* Open Global Styles and interact with the controls in the "Font Settings" panel. Publish the changes.
* Publish the post and check that the front-end also has the changes.
